tools/jbrowse/jbrowse_to_container.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
+<tool id="jbrowse_to_container" name="Add organisms to JBrowse container" version="@WRAPPER_VERSION@">
+    <description>will replace any existing organism</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="aggressive"><![CDATA[
+
+        if [ -z "@DATA_DIR@" ]; then
+            echo '@DATA_DIR@ variable must be set to use this tool';
+            exit 1;
+        fi
+
+        &&
+
+        rm -rf "@DATA_DIR@/"*
```

Comment:
Yes it is intended to rm everything, because:
-we rewrite the datasets.conf from scratch (with configfile)
-we don't want to have some already existing folders with the same names as the new datasets.

Maybe I could also just remove these problematic folders before doing the cp -R in the for loop, leaving the other folders untouched (but not accessible, as not listed in datasets.conf)?